package.json

```diff
@@ -16,6 +16,7 @@
     "@bufbuild/connect-web": "^0.13.0",
     "@bufbuild/protobuf": "^1.2.0",
     "@grpc/grpc-js": "^1.8.20",
+    "@openfeature/core": "^0.0.16",
     "@openfeature/web-sdk": "^0.4.0",
```

Comment:
```suggestion
```

Please move both of these to dev dependencies. (I realize one was not you, but it's confusing anyway so lets fix it).

Reply:
done.